tests/e2e/specs/amp-onboarding/template-mode-revisit.js

```diff
@@ -1,66 +0,0 @@
-/**
```

Comment:
Was deleting this intentional, or are the tests being handled somewhere else?

Reply:
See the PR description. I think it was intentional.

Reply:
It was intentional because they were brittle and were bound to cause headaches at some point. They're trying to do too much. I learned (or had the lesson reinforced) from these tests that e2e tests should go to a page, do something, test the result, and be done. Having multiple page navigations within a test was turning out to be problematic. I'll work on hardening the tests via unit testing and more discrete e2e tests in the next phase of work.

Reply:
Ah OK, thanks for the insight :+1:.